lib/modules/platform/gerrit/index.md

```diff
@@ -0,0 +1,65 @@
+# Gerrit
+
+## Authentication
+
+First, ![HTTP access token](/assets/images/gerrit-http-password.png) for the bot account.
+
+Let Renovate use your HTTP access token by doing _one_ of the following:
+
+- Set your HTTP access token as a `password` in your `config.js` file
+- Set your HTTP access token as an environment variable `RENOVATE_PASSWORD`
+- Set your HTTP access token when you run Renovate in the CLI with `--password=`
+
+Make sure this user is allowed to assign the Code-Review label with "+2" to his own changes or "automerge" can't work.
+
+Remember to set `platform=gerrit` somewhere in your Renovate config file.
+
+## Renovate PR/Branch-Model with Gerrit and needed Permissions
+
+If you use the "Code-Review" label and want `automerge` working, then you have to enable `gerritAutoApprove=true` in your Renovate config.
+In this case the bot will automatically add the _Code-Review_ label with the value "+2" to each created "pull-request" (Gerrit-Change).
+
+_Important: The login should be allowed to give +2 for the Code-Review label._
+
+**Also, you must use the following two settings. It will not work without these settings.**
+
+```
+{
+  platformCommit: true, //allow to reuse the Change-Id
+  gitNoVerify: ["push"], //allow-commit-hook to generate a Change-Id
+}
+```
+
+**Current Workarounds/Changes to git-based-commands:**
```

Comment:
> But perhaps the complete part can/should be removed from the documentation and moved to a new Refactor-issue?

Yeah, let's move this section to a new refactor issue for the Gerrit platform. I don't expect Renovate users must know about internal function names. 😉